cla_backend/libs/eligibility_calculator/tests/test_calculator.py

```diff
@@ -1975,24 +1959,6 @@ def test_cfe_with_incomplete_property_data_is_unknown(self):
         _, cfe_result = self.checker_with_blank_property()._do_cfe_civil_check()
         self.assertEqual("not_yet_known", cfe_result.overall_result)
 
-    def test_incomplete_income_data_is_unknown(self):
-        _, cfe_result = self.checker_with_income_without_earnings(maintenance_received=100,
-                                                                  child_benefits=500,
-                                                                  tax_credits=None)._do_cfe_civil_check()
-        self.assertEqual("not_yet_known", cfe_result.overall_result)
-
-    def test_incomplete_deductions_data_is_unknown(self):
-        _, cfe_result = self.checker_with_income_without_earnings(maintenance_received=100,
-                                                                  child_benefits=500,
-                                                                  deductions={})._do_cfe_civil_check()
-        self.assertEqual("not_yet_known", cfe_result.overall_result)
-
-    def test_incomplete_self_employment_is_unknown(self):
-        _, cfe_result = self.checker_with_income_without_earnings(maintenance_received=100,
-                                                                  child_benefits=500,
-                                                                  self_employed=True)._do_cfe_civil_check()
-        self.assertEqual("not_yet_known", cfe_result.overall_result)
-
```

Comment:
I've deleted these tests since they aren't realistic situations and unhelpful - if the income section is not complete then maintenance_received property doesn't exist, for example.

Reply:
tests added back now!